system/jlib/jstatcodes.h

```diff
@@ -233,6 +233,8 @@ enum StatisticKind
     StTimeBlocked,
     StCycleBlockedCycles,
     StCostExecute,
+    StTimeThorStartup,                  // Time taken for Thor instance to spin up before beginning to run query
```

Comment:
I'm not sure these should have the name startup in them.  Should it be TimeStartup (which can then be reused in other contexts)?